README.md

```diff
@@ -48,6 +48,10 @@ If you don’t plan on using the GitHub Action, you can directly apply your depl
 
 If you plan on deploying your application through your GitHub Action, commit all the files to your repository and watch your application get deployed!
 
+### `draft validate`
+
+Draft validate scans your manifests and populates warnings messages in your code terminal if your manifests are not following best practices. All best practices are sourced from AKS’s deployment safeguards feature. To learn more about deployment safeguards, visit the [offical documentation](https://learn.microsoft.com/azure/aks/deployment-safeguards). To view the best practices that are use in Draft, view the [safeguards package](https://github.com/Azure/draft/tree/main/pkg/safeguards/lib/v1.0.0).
```

Comment:
☕️👍 nice !! Might be worth it to fyi the release this has went out. Thanks for the ping.